### PR TITLE
Separated AvatarMixer inbound and outbound bandwidth on stats overlay

### PR DESCRIPTION
**Pull request by hyperlogic**
Also, small bug fix to BandwidthRecorder. It would incorrectly
calculate outbound pps if the input pps was 0.

### CODE REVIEW

File: libraries/networking/src/BandwidthRecorder.cpp [hunk 1]
```diff
@@ -27,9 +27,9 @@ float BandwidthRecorder::Channel::getAverageInputPacketsPerSecond() {
 }
 
 float BandwidthRecorder::Channel::getAverageOutputPacketsPerSecond() {
-    float delt = _input.getEventDeltaAverage();
+    float delt = _output.getEventDeltaAverage();
```
Comment (ZappoMan) [inline libraries/networking/src/BandwidthRecorder.cpp]: coding standard: variable names should be descriptive... what's a "delt"?

Comment (ZappoMan) [inline libraries/networking/src/BandwidthRecorder.cpp]: if you could fix the other function too that would be splendid!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0cebeb7fe9a6d3d917cdd3d491124f9937033cbd available at: https://gist.github.com/a2611243f366b1422cac<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6018.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6018.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 00f3b7b1b5ff7b5af0a6f450a16f9b0a9d29a77e available at: https://gist.github.com/a2611243f366b1422cac<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6018.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6018.dmg

**Pull request merged**